.circleci/config.yml

```diff
@@ -0,0 +1,115 @@
+# These environment variables must be set in CircleCI UI
+#
+# NPM_TOKEN - A valid NPM token for releases
+version: 2.1
+
+defaults: &defaults
+  working_directory: ~/addons-scanner-utils
+  docker:
+    # This is the NodeJS version we run in production.
+    - image: circleci/node:10
+
+defaults-next: &defaults-next
+  working_directory: ~/addons-scanner-utils
+  docker:
+    # This is the NodeJS version we run in production.
```

Comment:
Presumably we're not running both 10 and 12 at the same time?

Reply:
doh, wrong copy paste